src/locales/de.json

```diff
@@ -2,6 +2,8 @@
   "nav": {
     "home": "Home",
     "browse": "Stöbern",
+    "playlist": "Playliste",
```

Comment:
I'm not sure if you can translate it here. These values might be overridden with the english values again.